minard/templates/burst.html

```diff
@@ -61,30 +60,31 @@ <h2>Supernova Burst Trigger </h2>
             </div>
 
             <h3>Details</h3>
+            <small> <b>SH End:</b> 0=normal, 1=end of file reached, 2=events out of order, 3=too long, 4=clock jumps</small> <br>
```

Comment:
This font is really small for me, maybe try slightly size larger?

Reply:
I wanted this to be some form of hint/legend so smaller than the rest of the content. It's now slightly bigger.